torch/lib/c10d/TCPStore.cpp

```diff
@@ -1,33 +1,57 @@
 #include <c10d/TCPStore.hpp>
 
+#ifdef _WIN32
+#include <winsock2.h>
+#include <io.h>
+#else
 #include <poll.h>
-
 #include <unistd.h>
+#endif
+
 #include <algorithm>
+#include <fcntl.h>
 #include <system_error>
 
 namespace c10d {
 
 namespace {
 
-enum class QueryType : uint8_t { SET, GET, ADD, CHECK, WAIT, GETNUMKEYS, DELETE };
+enum class QueryType : uint8_t { SET, GET, ADD, CHECK, WAIT, GETNUMKEYS, DELETED };
```

Comment:
cc @osalpekar is this change OK?

Reply:
Yes it should work since the code has been changed in the right places (in `TCPStoreDaemon::query` and `TCPStore::deleteKey`). 

Curious what the motivation is behind the name change, though? These enum names are all actions on the store, so `DELETE` makes more sense.

Reply:
DELETE is a defined macro in winnt.h. So need change this enum name. How about change to DELETING? Or any other suggestion?

Reply:
Let's use `DELETE_KEY`